views/pages/GraphPage.jsx

```diff
@@ -269,10 +213,26 @@ class Graph extends Component {
       openStat: true,
     })
   }
+  fetchGraph = async (subject, study) => {
+    return await window
+      .fetch(`${basePath}/dashboard/${study}/${subject}`, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'same-origin',
+      })
+      .then((res) => res.json())
+      .then(({ data }) => {
+        this.setState({ graph: data.graph, subject: data.subject })
+      })
+  }
   componentDidUpdate() {}
   // eslint-disable-next-line react/no-deprecated
   async componentWillMount() {
     try {
+      const { subject, study } = this.props.params
```

Comment:
we're getting the subject from `state` above but `props.params` here. Which should we use?

Reply:
[b6abbbc](https://github.com/AMP-SCZ/dpdash/pull/319/commits/b6abbbc24c748cc38c461a04aec527463a8cb186)